float32 x
float32 y
float32 z
float32 az
float32 el
float32 ro


